### ISSUE TITLE
Pupillage field mandatory for all candidates.

### ISSUE THREAD

**Issue by joy-ade** [jac-uk/apply#790]
Its seem like a lot of candidate skip the pupillage questions. Can we make this field  mandatory for candidates.  This is having a big  impact when selection exercise download the eligibility issues report. The date for pupillage are  missing in the download![Capture.PNG48.PNG](https://images.zenhubusercontent.com/5f7b3a91edd7a0a890566450/3c16723f-c87e-43d2-9295-308a35cf1441)

**Comment by tomlovesgithub** [comment]
Hi Joy, it seems this particular issue is covered by the changes in #747. The field will be mandatory as part of those changes. 
Any details not found in a report or download i think should be placed under another ticket (or we could edit this one)

**Comment by KateMJAC** [comment]
This work is being covered by ticket #747. Once this work is complete you will not be able to save and continue withiout provividing a date to the questions 'Have you completed pupillage?' and 'Have you been called to the bar?'
